test/functional/scan/CMakeLists.txt

```diff
@@ -19,8 +19,12 @@ if(RAJA_ENABLE_HIP)
   list(APPEND SCAN_BACKENDS Hip)
 endif()
 
+if(RAJA_ENABLE_SYCL)
+  list(APPEND SCAN_BACKENDS Sycl)
+endif()
+
 
-set(SCAN_TYPES Exclusive ExclusiveInplace Inclusive InclusiveInplace)
+set(SCAN_TYPES ExclusiveInplace Exclusive  Inclusive InclusiveInplace)
```

Comment:
Why reorder these?

Reply:
I did that when I was testing because I commented the others out and only wanted to run exclusive inplace, but it's no longer needed. I reversed the order back to the original.